docs/ss_slide_002.md

```diff
@@ -18,4 +18,4 @@ Chan, Bert Wang-Chak. "Lenia-biology of artificial life." arXiv preprint arXiv:1
 
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_001.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_003a)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_001.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_003a)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.